pkg/unidling/controller/controller.go

```diff
@@ -325,7 +326,8 @@ func (c *UnidlingController) handleRequest(info types.NamespacedName, lastFired
 
 		scale.Spec.Replicas = scalableRef.Replicas
 
-		if err = scaleAnnotater.UpdateObjectScale(info.Namespace, scalableRef.CrossGroupObjectReference, obj, scale); err != nil {
+		codec := kapi.Codecs.LegacyCodec(registered.EnabledVersions()...)
```

Comment:
@liggitt Switched to use `runtime.Encode()`, however I am not sure if this is the right way of obtaining a codec here